src/test/Components.test.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import {configure, shallow} from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import {Counter} from "../components/Counter";
+import {DeleteButton} from "../components/DeleteButton";
+import ImageList from "../components/ImageList";
+import {ImageListContainer} from "../components/ImageListContainer";
+import {InputForm} from "../components/InputForm";
+import {ADD, END, START} from "../constants";
+import {addCat} from "../actions";
+
+configure({adapter: new Adapter()});
+
+
+describe("Counter", () => {
+    it("should render counter", () => {
+        const wrapper = shallow(<Counter count={1}/>);
+        expect(wrapper).toMatchSnapshot();
+    });
+});
+
+describe("Delete button render", () => {
+    it("should render Delete button", () => {
+        const deleteImages = jest.fn();
+        const wrapper = shallow(<DeleteButton countToDelete={1} deleteImages={deleteImages}/>);
+        expect(wrapper).toMatchSnapshot();
+    });
+});
+
+describe("Delete button onClick", () => {
+    it("should handle onClick in Delete button", () => {
+        const deleteImages = jest.fn();
+        const wrapper = shallow(<DeleteButton countToDelete={1} deleteImages={deleteImages}/>);
+        wrapper.find("button").simulate("click")
+        expect(deleteImages).toHaveBeenCalled();
+    });
+});
+
+describe("Image list", () => {
+    it("should render ImageList", () => {
+        const wrapper = shallow(<ImageList images={[]}/>);
```

Comment:
This test doesn't quite checks much with the empty `images`. So, let's prepare and pass some test data.